CONVERTER_SERVICE/README.md

```diff
@@ -29,4 +29,15 @@ ZIP_PATH
 npm install md5
 ```
 
+### for the program to run correctly
```

Comment:
Sugerencia para este titulo: _**Requirements to correctly run the program**_

Reply:
I agree, thank you!